src/main/java/io/druid/data/input/MapBasedRow.java

```diff
@@ -159,6 +161,52 @@ public long getLongMetric(String metric)
   }
 
   @Override
+  public int getIntMetric(String metric)
+  {
+    Object metricValue = event.get(metric);
+
+    if (metricValue == null) {
+      return 0;
+    }
+
+    if (metricValue instanceof Number) {
+      return ((Number) metricValue).intValue();
+    } else if (metricValue instanceof String) {
+      try {
+        return Integer.valueOf(((String) metricValue).replace(",", ""));
```

Comment:
replacing commas is not safe if you have numbers in a different locale, e.g. most european countries use `,` a as decimal point. I think we should limit parsing to what `parseInt` and the like do and not assume any country specific formatting.